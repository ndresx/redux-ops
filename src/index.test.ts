import * as index from './index';

describe('index', () => {
  it('should export', () => {
    expect(index).toMatchSnapshot();
  });
});
