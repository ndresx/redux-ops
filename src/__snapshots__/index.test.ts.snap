// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should export 1`] = `
Object {
  "BlueprintActionKey": Object {
    "Delete": "delete",
    "Error": "error",
    "Start": "start",
    "Success": "success",
  },
  "OpStatus": Object {
    "Error": "error",
    "Intermediate": "intermediate",
    "Started": "started",
    "Success": "success",
  },
  "actionTypes": Object {
    "BLUEPRINT": "@@redux-ops/BLUEPRINT",
    "DELETE": "@@redux-ops/DELETE",
    "RESET": "@@redux-ops/RESET",
    "START": "@@redux-ops/START",
    "UPDATE": "@@redux-ops/UPDATE",
    "prefix": "@@redux-ops/",
  },
  "actions": Object {
    "deleteOperation": [Function],
    "resetOperations": [Function],
    "startOperation": [Function],
    "updateOperation": [Function],
  },
  "createBlueprint": [Function],
  "createBlueprintActionTypes": [Function],
  "default": [Function],
  "defaultState": Object {},
  "getUniqueId": [Function],
  "opBroadcast": [Function],
  "opUnique": [Function],
  "opsMiddleware": [Function],
  "selectors": Object {
    "getOpById": [Function],
    "getOpByIds": [Function],
    "getOps": [Function],
    "getOpsState": [Function],
    "searchOps": [Function],
    "setReducerName": [Function],
  },
  "utils": Object {
    "createOperation": [Function],
    "getOpById": [Function],
    "getOpByIds": [Function],
    "searchOps": [Function],
  },
}
`;
